EIPS/eip-5920.md

```diff
@@ -43,10 +43,12 @@ Currently, to send ether to an address requires you to call into that address, w
 A new opcode is introduced: `PAY` (`0xfc`), which:
 
 - Pops two values from the stack: `addr` then `val`.
-- Charges the gas cost detailed below.
 - Exceptionally halts if `addr` has any of the high 12 bytes set to a non-zero value (i.e. it does not contain a 20-byte address).
-- Transfers `val` wei from the current target address to the address `addr`.
+- Charges the gas cost detailed below.
 - Marks `addr` as warm (adding `addr` to `accessed_addresses`).
+- Transfers `val` wei from the current target address to the address `addr`.
+- Pushes `1` to the stack if the transfer succeeds, or `0` if it fails.
```

Comment:
This is consistent with `*CALL` behavior in legacy EVM, but not with the planned `EXT*CALL` opcodes in EOF [see 7069](https://eips.ethereum.org/EIPS/eip-7069), which has richer codes and returns 0 on success.

Would PAY's return codes differ when called in EOF vs legacy?

Reply:
No, good point, I will change. But then I will mark this EIP as EOF only

Reply:
I forgot to do this, sorry. I have just pushed a commit which should reflect the changes in the way that they reflect what happens if you CALL with value and the value is more than the current balance OR similar for EXTCALL. Could you double check the values I push to stack?

Reply:
I have also decided against making it EOF-only. If we want that, then that should be a different PR.